functions/Get-DbaNetworkConfiguration.ps1

```diff
@@ -135,9 +136,13 @@ function Get-DbaNetworkConfiguration {
             if ($regRoot) {
                 $regPath = "Registry::HKEY_LOCAL_MACHINE\$regRoot\MSSQLServer\SuperSocketNetLib"
                 try {
-                    $forceEncryption = switch ((Get-ItemProperty -Path $regPath -Name ForceEncryption).ForceEncryption) { 0 { $false } 1 { $true } }
+                    $acceptedNtlmSpns = (Get-ItemProperty -Path $regPath -Name AcceptedSPNs).AcceptedSPNs
```

Comment:
I'd be prone to match the name of the registry key. I think those that work with the advanced settings that much in SQL Server are not going to notice the difference, and likely use the registry key more than SSCM to manage it.

Reply:
yeah i do like the name of the registry keys, they are descriptive

Reply:
Ok, changed AcceptedNtlmSpns to AcceptedSPNs.